README.md

```diff
@@ -140,6 +140,15 @@ EOF
 
 As you make local changes to the code, restart the operator to enact the changes.
 
+#### Test
+
+Before submitting PR, please your code. 
+Ensure there is a cluster running before running the testing script.
```

Comment:
Why is a cluster required? Isn't it just unit tests?

Reply:
yes, cluster test included in ./test/e2e folder.

Reply:
I split the non-cluster and cluster tests out to avoid confusion.